### ISSUE TITLE
Help with releasing via GitHub

### ISSUE THREAD

**Issue by andymeneely** [andymeneely/squib#187]
Lately I've been putting my game releases on Dropbox. This works, but I could also do releases via GitHub - especially since you can upload separate binaries.

There might not be an easy feature to add to Squib for this, but to start I should implement some Rake tasks that automate this for you.

https://developer.github.com/v3/repos/releases/#upload-a-release-asset

**Comment by andymeneely** [comment]
Looks like Travis and GitHub work nicely for this. I'm still refining my Travis file for this, but here's what I've got so far: 

https://github.com/andymeneely/victory-point-salad/blob/master/.travis.yml

This is more of a documentation thing than a Squib feature.

**Comment by bbugh** [comment]
GitHub actions can do this easily! No need for a third party system like Travis. For example: https://github.com/actions/create-release

**Comment by andymeneely** [comment]
Yeah I’ve mostly switched to GitHub actions for my own projects now. At some point I’ll document it.
